src/App.js

```diff
@@ -1,30 +1,43 @@
 import React from 'react';
-
+import PropTypes from 'prop-types';
 import './App.css';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { PostList } from './components/Postlist/PostList';
+
+const App = () => {
+  const postsInformation = posts.map(post => (
```

Comment:
name `postInformation` is for some object or primitive types,
if you have an array, it'll be better to name it something like `preparedPosts`

Reply:
countable plural form